### PR TITLE
Add raw option to display the raw byte count

### PR DESCRIPTION
**Pull request by andrewiggins**
For libraries that want to keep their size down, showing the raw byte size can be useful to evaluate exactly the size difference between implementations. The `--raw` option shows the raw byte size of all bundles.

Open other names as well.

### CODE REVIEW

**Comment by developit** [comment]
YUS

File: src/index.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ export default async function microbundle(options) {
 	}
 
 	function formatSize(size, filename, type) {
-		const pretty = prettyBytes(size);
+		const pretty = options.raw ? `${size} B` : prettyBytes(size);
```
Comment (developit) [inline src/index.js]: @andrewiggins I'm wondering if we could get around even having to add the `--raw` option here - what if we just always show bytes for <5000b libs?

```js
const pretty = size < 5000 ? `${size} B` : prettyBytes(size)
```

**Comment by kristoferbaxter** [comment]
Alternate proposal – always show the raw bytes in addition to the compressed resource sizes?

Good idea @andrewiggins.

File: src/index.js (hunk 1 above)
Comment (kristoferbaxter) [reply src/index.js]: In order to get this to work, you'll likely need to move the option read to `getSizeInfo` now.

Also, the size passed to this function is pre-compressed, so I'd recommend getting the size directly from `code` in the function below.

**Comment by Andarist** [comment]
Code-wise it's good. I'm wondering how useful this is at the moment though - what I mean is that `microbundle` doesn't make it easy now to compare previous & actual bundle sizes and you mention that this the main motivation behind this feature. Maybe we could do better by storing some snapshot file with those sizes so they would be easily comparable when viewing git diffs?

**Comment by developit** [comment]
@Andarist I've done that before by grabbing the file sizes prior to running builds, then calculating the differences post-build.

File: test/fixtures/raw/src/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+export const OneKilobyteOfBobRoss = `
+	We might as well make some Almighty mountains today as well, what the heck.
```
Comment (developit) [inline test/fixtures/raw/src/index.js]: this is my favorite new feature

**Comment by developit** [comment]
I've merged - only change we may want to make here is that it seems like the <5000b automatic raw mode should take effect based on gzipped size rather than original size.

**Pull request merged**